README.md

```diff
@@ -83,7 +83,7 @@ Additional warning messages may appear when including the monthly predefined rep
 This will create filename_out.json files when EnergyPlus is run at the command line. 
 
 Note: This utility was designed to work with files produced using EnergyPlus at the command line. Some file renaming might be necessary if using EP-Launch. 
-If using EP-Launch the eplusout.json and eplusout_hourly.json files may be found in the EPTEMP directory without the specific file name.
+If using EP-Launch, the eplusout.json and eplusout_hourly.json files may be found in the EPTEMP directory without the specific file name.
```

Comment:
Any thought about the new Python EP-Launch?  If we merge the big EP-Launch PR down and do a full release of it, would that make it more appropriate to mention here?

Reply:
Sure, that would be a good idea.